src/graphql/mutation/createMutationType.js

```diff
@@ -15,13 +15,13 @@ const createMutationType = schema =>
         schema
         .getProcedures()
         .filter(({ isMutation }) => isMutation)
-        .filter(procedure => !procedure.hasTableArg())
         .map(procedure => [procedure.getFieldName(), createProcedureMutationField(procedure)])
       ),
       // Add standard fields for tables.
       ...(
         schema
         .getTables()
+        .filter(table => table.kind === 'r')
```

Comment:
This would be a breaking change, so I’d rather not have it in the PR. Although I get your point about not allowing views that aren’t updatable to have mutations. I think the best solution is to use access levels to determine what relations (tables, views, etc) the user has been granted rights to mutate.